src/pages/mydashboard/components/SearchDashboard/index.tsx

```diff
@@ -0,0 +1,53 @@
+import styles from './SearchDashboard.module.scss';
+import Image from 'next/image';
+import { ChangeEvent, Dispatch, SetStateAction } from 'react';
+import { debounce } from 'lodash';
+
+type Invitation = {
+  id: number;
+  inviter: { nickname: string; email: string; id: number };
+  teamId: string;
+  dashboard: { title: string; id: number };
+  invitee: { nickname: string; email: string; id: number };
+  inviteAccepted: boolean;
+  createdAt: string;
+  updatedAt: string;
+};
+
+type SearchDashboardProps = {
+  initialInvitations: Invitation[];
+  setInvitations: Dispatch<SetStateAction<Invitation[]>>;
+};
+
+function SearchDashboard({
+  initialInvitations,
+  setInvitations,
+}: SearchDashboardProps) {
+  const debouncing = debounce((value: string) => {
+    const searchInvitations = initialInvitations.filter(
+      (invitation) =>
+        invitation.dashboard.title.includes(value) ||
+        invitation.inviter.nickname.includes(value),
+    );
+    setInvitations(searchInvitations);
+  }, 400);
+
+  const handleSubmit = (e: ChangeEvent<HTMLInputElement>) => {
+    debouncing(e.target.value);
+  };
```

Comment:
`onChange`의 핸들러로 `handleChange`가 더 적합해 보여요!
```suggestion
  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
    debouncing(e.target.value);
  };
```

Reply:
핸들러 이름은 이벤트의 의도를 파악할 수 있게 지어주는게 좋습니다!
handeChange, handleClick, handleSubmit과 같은 범용적인 이름은 의미를 유추하는게 힘들어질  수 있어요!